lib/apis/v3/policy.go

```diff
@@ -137,6 +137,9 @@ type EntityRule struct {
 	// For NetworkPolicy, an empty NamespaceSelector implies that the Selector is limited to selecting
 	// only workload endpoints in the same namespace as the NetworkPolicy.
 	//
+	// For NetworkPolicy, `global()` NamespaceSelector implies that the Selector is limited to selecting
+	// only GlobalNetworkSet or HostEndpoint.
```

Comment:
Not sure if this is right place to explain every possible selector expression.  Would be OK if it was a special case for just this field, but I don't believe that's the case - right?

Reply:
I think you're right, if you feel strongly about this, I can take it out, and port the changes over to the enterprise as well